Content.Client/HealthAnalyzer/UI/HealthAnalyzerWindow.xaml.cs

```diff
@@ -36,6 +36,9 @@ public void Populate(HealthAnalyzerScannedUserMessage msg)
 
                 text.Append($"{Loc.GetString("health-analyzer-window-entity-health-text", ("entityName", entityName))}\n");
 
+                text.Append(String.Format("Temperature: {0:F1} °C\n", msg.Temperature - 273f));
+                text.Append(String.Format("Blood Level: {0:F1} %\n", msg.BloodLevel * 100));
```

Comment:
1. Can you not get the TemperatureComponent and BloodstreamComponent here, just like `damageable` was obtained?
2. You need to add more localized strings. look where health-analyzer-window-entity-damage-total-text comes from (file search) and add "Blood Level: " there.